.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: "\U0001F4A1 new-feature"
+assignees: ''
+
+---
+
+<!--
+SPARKLERS BEWARE! THIS SPARKLETOWN/SPARKLE REPO IS PUBLIC TO THE ENTIRE INTERNET. ARE YOU SURE YOU WANT TO BE OPENING AN ISSUE HERE AND NOT IN SPARKLETOWN/INTERNAL-SPARKLE-ISSUES INSTEAD?
+-->
```

Comment:
[note] This is the only difference from the default GitHub issue templates the creation tool suggests, and is based off the comment GitHub themselves use to ensure people don't create issues in the wrong place ([ref](https://github.com/github/docs/edit/main/.github/ISSUE_TEMPLATE/improve-the-site.md))